src/test/chain-simulator/blocks.cs-e2e.ts

```diff
@@ -0,0 +1,186 @@
+import axios from 'axios';
+
+const CHAIN_SIMULATOR_URL = 'http://localhost:8085';
+const API_SERVICE_URL = 'http://localhost:3001';
+
+describe('Blocks e2e tests with chain simulator', () => {
+  beforeAll(async () => {
+    try {
```

Comment:
we could have a common function that first checks the epoch reached by CS and only if lower than 2, generate blocks until then. then we should call this function on each `beforeAll()`

Reply:
done